### ISSUE TITLE
not working?

### ISSUE THREAD

**Issue by cannap** [enniel/adonis-acl#3]
Hi i try to use acl but i have no luck

so what i do is

i create 2 Roles

test and administrator
now i assigned the administrator my user 
```js
Route.group(() => {
  Route.post('/category', 'CategoryController.store')
})
  .prefix('/api/v1/admin')
  .middleware(['auth:jwt', 'is:(test)']) //the user has no "test" role this is for testing but requests happens everytime
```
but when i make a request to this route its always posted 

role_user relations exists in the db this is for the administrator role
![alt](https://i.gyazo.com/eb595cd12432dea51d1c88c26eeb9f40.png)

this is how i assigne
```js
 const data = request.only([
      'username',
      'email',
      'password',
      'password_confirmation'
    ])

    const validation = await validate(data, User.storeRules)

    if (validation.fails()) {
      return response.status(400).send('nope...')
    }

    /**
     * Generate admin Role
     */
    const roleAdmin = new Role()
    roleAdmin.name = 'Administrator'
    roleAdmin.slug = 'administrator'
    roleAdmin.description = 'manage administration privileges'
    await roleAdmin.save()

    /**
     * Create admin User
     */
    delete data.password_confirmation
    const user = await User.create(data)
    await user.roles().attach([roleAdmin.id])
    response.json({ success: true })
```

i dont know whats wrong

**Comment by enniel** [comment]
Fixed in 1.0.2

**Comment by cannap** [comment]
works!

thanks
